modules/view/main/index.coffee

```diff
@@ -12,71 +17,77 @@ class View
   #XXX: Tidy and optimise
```

Comment:
idk, I'm not sure I particularly like the "Let's start with an empty array and push stuff into it" but I didn't want to modify too much without adding more tests